joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/ast/TypeDeclTests.scala

```diff
@@ -279,4 +279,105 @@ class TypeDeclTests extends CSharpCode2CpgFixture {
       }
     }
   }
+
+  "an anonymous type with primitive type members" should {
```

Comment:
In these tests, could you also check if the LHS of the assignment, e.g. `Foo` below, are assigned the type of the anonymous class?